app/models/manageiq/providers/amazon/agent_coordinator.rb

```diff
@@ -255,8 +255,8 @@ def deploy_agent
         }],
       ).first
     rescue Aws::EC2::Errors::InvalidParameterValue => e
-      if max_retries > 0
-        sleep 5
+      if max_retries.positive?
+        sleep 10
```

Comment:
I wouldn't change the sleep interval, just the retries. If the wait time is in the order of seconds, you want to wait less and retry more.

Reply:
Ok, changed sleep interval back to 5.